### PR TITLE
Fix publish package in github action

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -16,15 +16,15 @@ jobs:
         run: sh .github/scripts/check-release.sh
       - name: Install dependencies
         run: yarn && yarn install:functions
-      - name: Enter functions directory
-        run: cd functions
       - name: Publish with latest tag
         if: "!github.event.release.prerelease"
-        run: npm publish .
+        run:
+          npm publish functions
```
Comment (brunoocasali) [inline .github/workflows/publish.yml:22]: You could use `working-directory` once in the job definition, and all the commands will use it :)

**Review by brunoocasali — APPROVED** [review]
🔥 🚒

File: .github/workflows/publish.yml (hunk 1 above)
Comment (alallema) [reply .github/workflows/publish.yml:22]: Yes but it is for all the jobs in the same step and the `Check release validity` step need to be at the root

**Comment by alallema** [comment]
bors merge

**Pull request merged**